### ISSUE TITLE
Accessible example

### ISSUE THREAD

**Issue by paipai** [mozbrick/brick#90]
Your examples should be accessible. And be screen reader ready.
Replace these div with click events by anchor tags. It would be a good reason to use brick. 

This should be done a start of the project. This should be a prerequisite for any example.

**Comment by potch** [comment]
We're planning a full accessibility review before we release 1.0, including the fixes you mentioned. I agree that accessibility should be a first concern, but many of these components began as prototypes.

**Comment by potch** [comment]
I agree that accessibility should be a core tenet of software projects. We're definitely not there yet. There will be a full accessibility review of every component prior to a 1.0 launch. I've filed #91 to track that work. I'll close this up in favor of that tracking issue. Thanks for taking the time to file!

**Comment by paipai** [comment]
Thank you for the reply and for the great work.
